### PR TITLE
Day 3 Workflow

### LINKED ISSUE #32: Tie together the Navigation flow for Day 3
Implement the navigation flow for these set of events on Day 3:
- [ ]  Episode Recall
    - [ ]  List Episodes from Day 1
    - [ ]  Recall Random Episode from Day 1
    - [ ]  Recall Random Episode from Day 1
    - [ ]  List Episodes from Day 2
- [ ]  Create Episodes for the Day
- [ ]  Record Episodes for the Day

This may be a big issue since the episode recall may not be fully-fledged out with this yet. If it grows too large a PR, I can break this issue down into finishing the recall for Day 3 and tieing together the navigation.

### PR DESCRIPTION
**Pull request by zefwang**
Adding the nav flow for day 3.

### CODE REVIEW

**Review by dfarooq610 — APPROVED** [review]
I think it all looks good. 

I have one question: where do you think is a good place to upload the JSON file for Day 3? On main, we currently upload info on the episode prediction screen. There is a helper function in the EpisodePredictionScreen named uploadEpisodeInfo or something like that that you can use. Perhaps the upload can happen in the background of the final Thank You Screen?

File: components/EpisodeListingOverviewDay3.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      <Text style={episodeListingStyles.subheader}>Let’s start recording</Text>
+      <Text style={episodeListingStyles.text}>
+        For this first recording, we would like you to please tell us the names
+        of all of your Episodes from Day 2. {"\n\n"}If you cannot remember the
+        exact title, you can try to identify the episode by a short, few word
+        description. {"\n\n"}
+        <Text style={episodeListingStyles.doNot}>Do not</Text> tell us any
+        episodes from today, only from Day 2.
+      </Text>
+    </View>
```
Comment (jrosen081) [inline components/EpisodeListingOverviewDay3.tsx]: This seems to be the same (other than the day #) as the EpisodeListing for day 2. Should we combine these two?

**Review by jrosen081 — APPROVED** [review]
Legendary, golden, touching, momentous. That is how I feel about this or!

File: components/EpisodeListingOverviewDay3.tsx (hunk 1 above)
Comment (zefwang) [reply components/EpisodeListingOverviewDay3.tsx]: This is a very valid point and we have removed this file, adding a prop to the EpisodeListingOverview component.

**Pull request merged**